src/main/java/Freedom.java

```diff
@@ -0,0 +1,96 @@
+import java.util.Scanner;
+
+public class Freedom {
+    protected static Task[] storage = new Task[100];
```

Comment:
I like that you attached the array specifier `[]` to the array type `Task` and not the array variable `storage`!